internal/envoy/cluster.go

```diff
@@ -106,6 +114,36 @@ func cluster(cluster *dag.Cluster, service *dag.TCPService) *v2.Cluster {
 	return c
 }
 
+func ClusterLoadAssignConfig(cluster string, service *dag.TCPService) *v2.ClusterLoadAssignment {
```

Comment:
// StaticClusterLoadAssignment creates a *v2.ClusterLoadAssignment pointing to the external DNS address of the service.